test/strategy/ExtendConfigStrategyTest.js

```diff
@@ -7,6 +7,43 @@ var strategy = require('../../lib/strategy');
 var ExtendConfigStrategy = strategy.ExtendConfigStrategy;
 
 describe('ExtendConfigStrategy', function () {
+  it("should be able to extend member that references itself", function (done) {
+    var extendConfig = {
+      abc: "123"
```

Comment:
Yes, using `const`. Should be any problems since we're using node and this is just in the tests, right?